src/gordon_janitor_gcp/clients/_utils.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should we name a module that will be imported with a leading underscore?  I thought the underscore was to indicate it was private[-ish]?

Reply:
wow delayed response! dang, how did I miss this.

You're right that it's to denote it's not a public module. But more so from the perspective of depending packages, i.e. in extending `gordon_janitor_gcp` for our Spotify-specific stuff, we should not import/use this `_utils` module. It's meant to only be called/used internally within this package (much like how semi private helper functions can still be called from public ones, just not externally). Does that make sense?